src/disco/components/AddonCompatibilityError/index.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable react/no-danger */
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import {
+  INCOMPATIBLE_FIREFOX_FOR_IOS,
+  INCOMPATIBLE_NOT_FIREFOX,
+  INCOMPATIBLE_UNDER_MIN_VERSION,
+} from 'core/constants';
+import _log from 'core/logger';
+import translate from 'core/i18n/translate';
+import { sanitizeHTML } from 'core/utils';
+
+import './style.scss';
+
+
+// Messages in the disco pane are a bit less specific as we expect fewer
+// non-Firefox clients and the copy space is limited.
+export class AddonCompatibilityErrorBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+    lang: PropTypes.string.isRequired,
+    log: PropTypes.object,
+    minVersion: PropTypes.string.isRequired,
+    reason: PropTypes.string.isRequired,
+  }
+
+  static defaultProps = {
+    log: _log,
+  }
+
+  render() {
+    const {
+      i18n,
+      lang,
+      log,
+      minVersion,
+      reason,
+    } = this.props;
+    const downloadUrl = `https://www.mozilla.org/${lang}/firefox/`;
+    let message;
+
+    if (typeof reason === 'undefined') {
+      throw new Error('AddonCompatibilityError requires a "reason" prop');
+    }
+    if (typeof minVersion === 'undefined') {
+      throw new Error('minVersion is required; it cannot be undefined');
+    }
+
+    if (reason === INCOMPATIBLE_NOT_FIREFOX) {
```

Comment:
Should never really happen since non non-FF browsers should ever see the disco pane.